### PR TITLE
👮 Control which users can have tickets assigned to them with a permission

### PR DESCRIPTION
**Pull request by rowanseymour**
Currently it's just admins + agents but seems like editors should be able to too

### CODE REVIEW

**Comment by norkans7** [comment]
editors have always been like admins except managing accounts and subscription/credits

**Comment by rowanseymour** [comment]
@norkans7 we've been telling users that for a long time but it's never been quite true - if you login in as an editor you'll see they don't have access to almost all org administration functionality. But they should certainly have access to tickets.

**Review by norkans7 — APPROVED** [review]

File: temba/tickets/views.py [hunk 1]
```diff
@@ -186,31 +187,26 @@ def form_valid(self, form):
...
-                required=True,
+            assignee = forms.ModelChoiceField(
+                queryset=User.objects.none(),
                 widget=SelectWidget(
                     attrs={
                         "searchable": True,
                         "widget_only": True,
                     }
                 ),
+                empty_label=_("Unassigned"),
```
Comment (ericnewcomer) [inline temba/tickets/views.py:198]: How can a ticket be unassigned now?

Comment (rowanseymour) [reply temba/tickets/views.py:198]: oh doh I think I need `required=False` here... well bug no. 1 found

**Pull request merged**